src/app/qgsdevtoolspanelwidget.cpp

```diff
@@ -43,6 +49,21 @@ QgsDevToolsPanelWidget::QgsDevToolsPanelWidget( const QList<QgsDevToolWidgetFact
 
 QgsDevToolsPanelWidget::~QgsDevToolsPanelWidget() = default;
 
+void QgsDevToolsPanelWidget::addToolWidget( QgsDevToolWidget *widget )
+{
+  mStackedWidget->addWidget( widget );
+
+  QListWidgetItem *item = new QListWidgetItem( widget->windowIcon(), QString() );
+  item->setToolTip( widget->windowTitle() );
+  item->setData( Qt::UserRole, widget->windowTitle() );
```

Comment:
```suggestion
  item->setData( Qt::UserRole, widget->objectName() );
```

Otherwise we're potentially using a translated value, and won't be able to hardcode in an identifier for the documentation browser item (see below)